concepticondata/conceptlists/Carling-2017-240c.tsv

```diff
@@ -1,241 +1,240 @@
+Carling-2017-240c-60	60	1300	COUNTRY	country, land	12219	Physical/natural world	
+Carling-2017-240c-61	61	623	LAGOON	lagoon	12220	Physical/natural world	
+Carling-2017-240c-62	62		*CLAY OR MUD	clay, mud	12221	Physical/natural world	
+Carling-2017-240c-63	63	1598	NORTH	north	12222	Physical/natural world	
+Carling-2017-240c-64	64	1595	SOUTH	south	12223	Physical/natural world	
+Carling-2017-240c-65	65	1877	EAST	east	12224	Physical/natural world	
+Carling-2017-240c-66	66	1817	WEST	west	12225	Physical/natural world	
+Carling-2017-240c-67	67	630	LIGHT (RADIATION)	light (n)	12226	Physical/natural world	
+Carling-2017-240c-68	68	11	DARKNESS	darkness	12227	Physical/natural world	
+Carling-2017-240c-69	69	943	WATERFALL	waterfall	12228	Physical/natural world	
+Carling-2017-240c-70	70	1732	SKY	sky	12229	Physical/natural world	
+Carling-2017-240c-71	71	960	WIND	wind	12230	Physical/natural world	
+Carling-2017-240c-72	72	611	HARVEST	harvest, crop	12140	Agriculture	
+Carling-2017-240c-73	73	714	SEED	seed (n)	12178	Agriculture	
+Carling-2017-240c-74	74	212	FIELD	cultivated field	12179	Agriculture	
+Carling-2017-240c-75	75	1541	CULTIVATE	to cultivate	12180	Agriculture	
+Carling-2017-240c-76	76	748	SOW SEEDS	to sow	12181	Agriculture	
+Carling-2017-240c-77	77	1927	BAMBOO	bamboo	12360	Wild plants	
+Carling-2017-240c-78	78			açaí berry	12361	Wild plants	
+Carling-2017-240c-79	79			moriche fruit	12362	Wild plants	
+Carling-2017-240c-80	80			genipap fruit	12363	Wild plants	
+Carling-2017-240c-81	81	239	FLOWER	flower	12364	Wild plants	
+Carling-2017-240c-82	82	1507	FRUIT	fruit	12365	Wild plants	
+Carling-2017-240c-83	83			patawa fruit	12366	Wild plants	
+Carling-2017-240c-84	84	677	AXE	axe	12193	Hunting and fishing	
+Carling-2017-240c-85	85	387	SLING	sling	12194	Hunting and fishing	
+Carling-2017-240c-86	86	994	BOW	bow	12195	Hunting and fishing	
+Carling-2017-240c-87	87	977	ARROW	arrow	12196	Hunting and fishing	
+Carling-2017-240c-88	88	945	SPEAR	spear	12197	Hunting and fishing	
+Carling-2017-240c-89	89	1074	FISHHOOK	fish-hook	12198	Hunting and fishing	
+Carling-2017-240c-90	90	388	FISHNET	fish-net	12199	Hunting and fishing	
+Carling-2017-240c-91	91	389	FISH TRAP	fish-trap	12200	Hunting and fishing	
+Carling-2017-240c-92	92	1435	HUNT	to hunt	12201	Hunting and fishing	
+Carling-2017-240c-93	93	1172	SHOOT	to shoot	12202	Hunting and fishing	
+Carling-2017-240c-94	94	1763	CLUB	club	12203	Hunting and fishing	
+Carling-2017-240c-95	95	1352	KNIFE	knife	12204	Hunting and fishing	
+Carling-2017-240c-96	96	1594	FLOUR	flour	12231	Products	
+Carling-2017-240c-97	97	1274	SALT	salt	12232	Products	
+Carling-2017-240c-98	98	404	ALCOHOL (FERMENTED DRINK)	fermented drink	12233	Products	
+Carling-2017-240c-99	99		*COOKING VESSEL	cooking vessel	12189	Food preparation	
+Carling-2017-240c-100	100	225	PESTLE	pestle, pounder	12190	Food preparation	
+Carling-2017-240c-101	101		*CUP OR GLASS	cup, glass	12191	Food preparation	
```

Comment:
I'd prefer something like "drinking vessel" (Trinkgefäß), but I'd also check with "cup". Glass is a specifier, and it is not meant to reflect the substance, you could simply link to "cup". And this would be enough, I'd say.